### PR TITLE
Updating some directives

### PR DESCRIPTION
**Pull request by elylucas**

### CODE REVIEW

File: docs/guides/core-concepts/writing-and-organizing-tests.mdx [hunk 1]
```diff
@@ -292,7 +292,18 @@ is set to look for one of the following files:
...
+
+<strong><Icon name="cogs"></Icon> supportFile per testing
+type</strong>
+
+Depending on which [testing type](/guides/core-concepts/testing-types)
+you are using, you can configure yoursupportFile accordingly.
+
+- [component](/guides/references/configuration#component)
+- [e2e](/guides/references/configuration#e2e)
+
```
Comment (jaffrepaul) [inline docs/guides/core-concepts/writing-and-organizing-tests.mdx:305]: Couple of tweaks. I'm not sure if we'd need to capitalize the types in this context but I believe that's an initiative on the PMM side. 

```suggestion
you are using, you can configure your `supportFile` accordingly.

- [Component](/guides/references/configuration#component)
- [E2E](/guides/references/configuration#e2e)

```

**Review by jaffrepaul — CHANGES_REQUESTED** [review]
Added a couple of small tweaks 👍

File: docs/guides/end-to-end-testing/testing-strategies/google-authentication.mdx [hunk 2]
```diff
@@ -107,23 +107,25 @@ Cypress to use the [Google](https://google.com) environment variables set in the
...
+GOOGLE_REFRESH_TOKEN = 'your-refresh-token'
 ```
 
-:::cypress-env-example
+:::cypress-config-example
 
 ```js
 // Populate process.env with values from .env file
-require("dotenv").config();
+require('dotenv').config()
```
[the remark this answers is not in the document]
Comment (elylucas) [reply docs/guides/end-to-end-testing/testing-strategies/google-authentication.mdx:119]: Its out everywhere else, so I hope so. Having versioned docs made this needed.

**Review by jaffrepaul — APPROVED** [review]
LGTM 🎉

**Pull request merged**